app/controllers/api/task_controller.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Api
+  class TaskController < Api::ApplicationController
```

Comment:
By convention we name controllers in plural
```suggestion
  class TasksController < Api::ApplicationController
```